docs/source/model_design/risk_factors.rst

```diff
@@ -0,0 +1,94 @@
+Relative Risk (RR)
+++++++++++++++++++
+
+Population Attributable Fraction (PAF)
+++++++++++++++++++++++++++++++++++++++
+
+Description
++++++++++++
+
+PAF in GBD 2017 (operationally)
++++++++++++++++++++++++++++++++
+
+Defintion of PAF
+++++++++++++++++
+
+The population attributable fraction (PAF) represents the proportion of risk that would be reduced in a given year if the exposure to a risk factor in the past were reduced to an ideal exposure scenario. This is defined for a continuous risk factor as:
```

Comment:
This could probably be defined for other time periods, other than a year, right? (Though 'yearly' I think is the framework that GBD uses)